hadoop-ozone/ozone-manager/src/main/resources/webapps/ozoneManager/ozoneManager.js

```diff
@@ -113,5 +113,36 @@
         require: {
             overview: "^overview"
         },
+        controller: function ($http) {
+            var ctrl = this;
+            $http.get("jmx?qry=Ratis:service=RaftServer,group=*,id=*")
+                .then(function (result) {
+                    ctrl.role = result.data.beans[0];
+                });
+
+            $http.get("jmx?qry=ratis:name=ratis.leader_election.*electionCount")
+                .then(function (result) {
+                    ctrl.electionCount = result.data.beans[0];
+                });
+
+            $http.get("jmx?qry=ratis:name=ratis.leader_election.*lastLeaderElectionElapsedTime")
+                .then(function (result) {
+                    ctrl.elapsedTime = result.data.beans[0];
+                    ctrl.elapsedTime.Value = convertMsToTime(ctrl.elapsedTime.Value);
+                });
+        }
     });
+        function convertMsToTime(ms) {
+          if(ms == -1){
```

Comment:
Is this a valid case? ms==-1 or should we remove it?

Reply:
Yes I have addressed the issue!
thanks !!